Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -596,7 +596,8 @@ WebPage::WebPage(PageIdentifier pageID, WebPageCreationParameters&& parameters)
 
     if (shouldBlockIOKit) {
 #if HAVE(SANDBOX_STATE_FLAGS)
-        CFPreferencesGetAppIntegerValue(CFSTR("key"), CFSTR("com.apple.WebKit.WebContent.BlockIOKitInWebContentSandbox"), nullptr);
+        auto auditToken = WebProcess::singleton().auditTokenForSelf();
+        sandbox_enable_state_flag("BlockIOKitInWebContentSandbox", *auditToken);
```

Comment:
Should we null check `auditToken`?

Reply:
> Should we null check `auditToken`?

That is a good point. In this case, we depend on always having an audit token, so a check may not be required here.

Thanks for reviewing!